docs/docker.rst

```diff
@@ -40,6 +34,9 @@ tag will always point to the latest git commit. All tagged versions of Faucet
 in git are also available to use, for example using the faucet/faucet:v1_3
 Docker will run the stable version 1.3 of Faucet.
 
+By default the Faucet and Gauge images are run as the `faucet` user under UID 0,
```

Comment:
I think this documentation is wrong, we should use ``-e LOCAL_USER_ID`` right?

-u doesn't work

> brad@nix/50:~/github/faucet$ docker run -it -u 100000 --rm -v /scratch/faucet_configs/:/etc/ryu/faucet/ faucet/faucet  
> Starting with UID : 0
> su-exec: setgroups(0): Operation not permitted

-e works:

> brad@nix/50:~/github/faucet$ sudo docker run -it -e LOCAL_USER_ID=100000 --rm -v /scratch/faucet_configs/:/etc/ryu/faucet/  faucet/faucet
> Starting with UID : 100000
> loading app faucet.faucet

Reply:
ah, yes. my bad. corrected.